packages/app/src/common/stacks-utils.ts

```diff
@@ -35,23 +43,105 @@ export const encodeContractCallArgument = ({ type, value }: ContractCallArgument
   }
 };
 
-export const getRPCClient = () => {
-  const { origin } = location;
-  const url = origin.includes('localhost')
-    ? 'http://localhost:3999'
-    : 'https://stacks-node-api.blockstack.org';
-  return new RPCClient(url);
-};
-
 export const stacksValue = ({
   value,
   fixedDecimals = false,
+  withTicker = true,
 }: {
-  value: number;
+  value: number | string;
   fixedDecimals?: boolean;
+  withTicker?: boolean;
 }) => {
-  const microStacks = new BigNumber(value);
-  const stacks = microStacks.shiftedBy(-6);
+  const stacks = microStxToStx(value);
   const stxString = fixedDecimals ? stacks.toFormat(6) : stacks.decimalPlaces(6).toFormat();
-  return `${stxString} STX`;
+  return `${stxString}${withTicker ? ' STX' : ''}`;
+};
+
+export const microStxToStx = (mStx: number | string) => {
+  const microStacks = new BigNumber(mStx);
+  const stacks = microStacks.shiftedBy(-6);
+  return stacks;
+};
+
+export const stxToMicroStx = (stx: number | string) => {
+  const stxBN = new BigNumber(stx);
+  const micro = stxBN.shiftedBy(6);
+  return micro;
 };
+
+export function shortenHex(hex: string, length = 4) {
+  return `${hex.substring(0, length + 2)}…${hex.substring(hex.length - length)}`;
+}
+
+/**
+ * truncateMiddle
+ *
+ * @param {string} input - the string to truncate
+ * @param {number} offset - the number of chars to keep on either end
+ */
+export const truncateMiddle = (input: string, offset = 5): string => {
```

Comment:
many of these methods exist in `@stacks/ui-utils`

Reply:
👍🏼 removed this - some stuff like `microStxToStx` return `BigNumber` - we should definitely get consistent with these. Most were written before being added to `@stacks/ui-utils`.